presto-hive/pom.xml

```diff
@@ -53,6 +53,12 @@
             <version>${dep.hudi.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.delta</groupId>
+            <artifactId>delta-presto_2.12</artifactId>
+            <version>0.2.1-SNAPSHOT</version>
+        </dependency>
```

Comment:
There are related test failures. Please fix them. Thanks

Reply:
Hi @v-jizhang  I am still working on artifact release. Will update soon and sent for review.